webapp/graphite/cache_metrics/views.py

```diff
@@ -0,0 +1,16 @@
+from graphite.compat import HttpResponse
+from graphite.util import json
+from graphite.carbonlink import CarbonLink
+
+
+def cache_metric(request):
+	queryParams = request.GET.copy()
```

Comment:
is this copy needed? it's an additional allocation that you don't really need to do, unless you're mutating data.